docs/plugins/discovery-gce.asciidoc

```diff
@@ -56,8 +66,13 @@ The following gce settings (prefixed with `cloud.gce`) are supported:
 
  `max_wait`::
 
-     The maximum elapsed time in milliseconds after the client instantiating retry. If the time elapsed goes past the
-     `max_wait`, client stops to retry. Defaults to 15 minutes (900000 milliseconds).
+     The maximum elapsed time after the client instantiating retry. If the time elapsed goes past the
+     `max_wait`, client stops to retry. A negative value means that it will wait indefinitely. Defaults to `-1s`.
```

Comment:
WDYM? Is this a problem of wording here or a problem with the code itself? Like we should remove `retry` and only keep `max_wait` (and rename it to `max_retry` for example)?
If this is what you are thinking about, I'm totally +1 but I think it should come in another PR.